loki/README.md

```diff
@@ -172,12 +135,88 @@ kubectl -n ${KYMA_NS} get pod -l app=loki,release=${HELM_LOKI_RELEASE}
      'time={NANOSECONDS}'
    ```
 
+## Install Grafana
+
+Because Grafana provides a very good Loki integration, you might want to install it as well.
+
+### Installation
+
+1. To deploy Grafana, run:
+
+   ```bash
+   helm upgrade --install --create-namespace -n ${KYMA_NS} grafana grafana/grafana -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/grafana-values.yaml
+   ```
+
+1. To enable Loki as Grafana Datasource, run:
+   ```bash
+   cat <<EOF | kubectl apply -n ${KYMA_NS} -f -
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: grafana-loki-datasource
+     labels:
+       grafana_datasource: "1"
+   data:
+      loki-datasource.yaml: |-
+         apiVersion: 1
+         datasources:
+         - name: Loki
+           type: loki
+           access: proxy
+           url: "http://${HELM_LOKI_RELEASE}:3100"
+           version: 1
+           isDefault: false
+           jsonData: {}
+   EOF
+   ```
+
+1. To access the Grafana UI with kubectl port forwarding, run:
+
+   ```bash
+   kubectl get secret --namespace mlp grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+   ```
+   
+   to get the password for the access, then run:
+   ```bash
+   kubectl -n ${KYMA_NS} port-forward svc/grafana 3000:80
+   ```
+   
+   Open Grafana in your browser under `http://localhost:3000` and log in with user admin and the password taken from the previous Helm command.
+  
+## Exposure
+1. To expose Grafana using the Kyma API Gateway, download the APIRule file and replace the `{release-name}` variable with the name of Helm release:
+   ```bash
+   curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/apirule.yaml -o apirule.yaml
+   ```
+1. Create an APIRule:
+   ```bash
+   kubectl -n ${KYMA_NS} apply -f apirule.yaml 
+   ```
+1. Get the public URL of your Loki instance:
+   ```bash
+   kubectl -n ${KYMA_NS} get vs -l apirule.gateway.kyma-project.io/v1beta1=grafana.${KYMA_NS} -ojsonpath='{.items[*].spec.hosts[*]}'
+   ```
+
+### Add a Link for Grafana to the Kyma Dashboard
+1. Download the `dashboard-configmap.yaml` file and change `{GRAFANA_LINK}` to the text you retrieved in the previous step.
+   ```bash
+      curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/dashboard-configmap.yaml -o dashboard-configmap.yaml
+   ```
+   You can change the label field to change the name of the tab, and the category tab if you wish to move it to another category.
+
+1. Apply the ConfigMap, and go to Kyma Dashboard. You should see a Link to the newly exposed Grafana under the Observability section. If you already have a busola-config, merge it with the existing one:
+   ```bash
+   kubectl apply -f dashboard-configmap.yaml 
+   ```
+   
```

Comment:
How do I see I was successful? Seeing the link is already part of the step instruction, so what comes afterwards to show me I was successful?